Author: Sean Finney <seanius@debian.org>
Description: Fix FTBFS: "gconf/gconf-client.h: No such file or directory"
 It seems that @GNOME_WINDOW_SETTINGS_LIBS@ used to implicitly include
 gconf-related compiler flags, but not always.  This should explicitly
 include them.
Thanks: Sebastian Harl <tokkee@debian.org>
Origin: vendor, debian
Bug-Debian: http://bugs.debian.org/562027
Bug-Debian: http://bugs.debian.org/562438
--- compiz.orig/gtk/gnome/Makefile.am
+++ compiz/gtk/gnome/Makefile.am
@@ -2,7 +2,8 @@ moduledir = $(DESTDIR)$(windowsettingsli
 
 if USE_GNOME
 libcompiz_la_LDFLAGS = -export-dynamic -avoid-version -rpath $(moduledir)
-libcompiz_la_LIBADD  = @GNOME_WINDOW_SETTINGS_LIBS@
+libcompiz_la_LIBADD  = @GNOME_WINDOW_SETTINGS_LIBS@ @GCONF_LIBS@
+libcompiz_la_CFLAGS  = @GCONF_CFLAGS@
 libcompiz_la_SOURCES = 		\
 	compiz-window-manager.c	\
 	compiz-window-manager.h
